### PR TITLE
{mpi}[gompi/2019b] OpenCoarrays v2.8.0

### PR DESCRIPTION
**Pull request by furstj**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/o/OpenCoarrays/OpenCoarrays-2.8.0-gompi-2019b.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+the coarray Fortran (CAF) parallel programming features of the Fortran 2008
+standard and several features proposed for Fortran 2015 in the draft Technical
+Specification TS 18508 Additional Parallel Features in Fortran."""
+
+toolchain = {'name': 'gompi', 'version': '2019b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/sourceryinstitute/opencoarrays/releases/download/%(version)s']
+sources = [SOURCE_TAR_GZ]
+patches = ['OpenCoarrays-2.2.0-cafrun-OpenMPI-mpiexec.patch']
```
Comment (verdurin) [inline easybuild/easyconfigs/o/OpenCoarrays/OpenCoarrays-2.8.0-gompi-2019b.eb]: @furstj Please use the macro here, as in previous versions.

Comment (boegel) [reply easybuild/easyconfigs/o/OpenCoarrays/OpenCoarrays-2.8.0-gompi-2019b.eb]: @verdurin An existing patch that was created for version 2.2.0 is being used here, so `%(version)s` can't be used here...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/fcdfe754fd8062def8244a86e813a1a3 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3133.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/8d8b8a636cbca344857932fac5e8e1c0 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2494.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/38ab2ef8133411c1897ab37a421d8fe9 for a full test report.

File: easybuild/easyconfigs/o/OpenCoarrays/OpenCoarrays-2.8.0-gompi-2019b.eb (hunk 1 above)
Comment (verdurin) [reply easybuild/easyconfigs/o/OpenCoarrays/OpenCoarrays-2.8.0-gompi-2019b.eb]: Ah, okay. That explains it. Thanks @boegel

**Comment by verdurin** [comment]
Going in, thanks @furstj!

**Pull request merged**